### ISSUE TITLE
Error while adding Dropout Layer or BatchNorm layer to the model.

### ISSUE THREAD

**Issue by raginisharma14** [fchollet/keras#7720]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [ ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [ ] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by raginisharma14** [comment]
I see the below error: InvalidArgumentError (see above for traceback): You must feed a value for placeholder tensor 'conv1_
1bn/keras_learning_phase' with dtype bool
[[Node: conv1_bn/keras_learning_phase = Placeholder[dtype=DT_BOOL, shape=<unknown>, _device
="/job:localhost/replica:0/task:0/cpu:0"]()]]
I have set the learning_phase to 1(K.set_learning_phase(1)). But still I am facing this issue.

**Comment by raginisharma14** [comment]
This issue is resolved by setting k.set_learning_phase(True) just before building the model.
